src/components/top-bar/_top-bar.scss

```diff
@@ -45,6 +51,11 @@
       }
     }
   }
+
+  .left-controls *,
+  .right-controls * {
+    pointer-events: auto;
```

Comment:
the top-bar itself it also a container so add this in it

Reply:
with scss you can do like:
```
  &.disabled {
    pointer-events: none;
    * {
       pointer-events: auto;
    }
  }```

Reply:
We tried first what you suggested but it doesn't fix the issue correctly. The problem is that the left and right controller also have dimensions due to its children, so it is also grabbing the clicks. That is why we lowered it another level